### PR TITLE
Gb12 general settings

### PR DESCRIPTION
**Pull request by salac-kolosek**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -55,7 +53,7 @@
           </div>
           <ul class="nav-list">
             <li class="list-item">
-              <a href="/stories/new" class="<%= active_left_nav(request.original_url, 'stories/new') %>">
+              <a href="/stories/new" class="<%= active_left_nav(request.original_url, '/stories/new') %>">
```
Comment (kris-kolosek) [inline app/views/layouts/application.html.erb]: Update the links with rails syntax "link_to" 
eg.: <%= link_to "New story", new_stories_path %>

File: app/views/overview/index.html.erb [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 		<div class="row margin-top-small	">
 			<% stories_row.each do |story| %>
 				<div class="story-overview">
 					<%= image_tag "blogging.jpg", size: "300x150", class: "margin-bottom-medium"%><br/>
 					<%= link_to story.title, story_path(story), class: "title" %>
-					<div class="content margin-top-small"><%= story.content %></div>
+					<div class="content margin-top-small"><%= raw story.markdown %></div>
 				</div>
 			<% end %>
 		</div>
```
Comment (kris-kolosek) [inline app/views/overview/index.html.erb]: Fix the spacing please.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (salac-kolosek) [reply app/views/layouts/application.html.erb]: Ok.

File: app/views/overview/index.html.erb (hunk 2 above)
Comment (salac-kolosek) [reply app/views/overview/index.html.erb]: Ok.

File: app/controllers/sites_controller.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+class SitesController < ApplicationController
```
Comment (kolosek) [inline app/controllers/sites_controller.rb]: Whole need for Sites - even though we will have only one Site, ever, is fishy.

Comment (kolosek) [reply app/controllers/sites_controller.rb]: @kris-kolosek DId you catch this one?

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -6,4 +6,12 @@ def active_left_nav(url, location)
       "listlink"
     end
   end
+
+  def get_protocol
+    Rails.env.production? ? "https://" : "http://"
+  end
+
+  def get_host
+    ENV['GHOST_BLOG_HOST']
```
Comment (kolosek) [inline app/helpers/application_helper.rb]: Is this the place for ENV variables or config?

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -76,16 +74,25 @@
             <% end %>
           </ul>
 
+          <% if current_user.admin? %>
+            <ul class="nav-list margin-top-small">
+              <li class="list-item">
+                <a href="/settings/general" class="<%= active_left_nav(request.original_url, '/settings/general') %>">
```
Comment (kolosek) [inline app/views/layouts/application.html.erb]: Didn't we say to use Rails links?

File: app/helpers/application_helper.rb (hunk 4 above)
Comment (salac-kolosek) [reply app/helpers/application_helper.rb]: I will move it to config.

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (salac-kolosek) [reply app/views/layouts/application.html.erb]: I will fix it.

**Pull request merged**